checks.py

```diff
@@ -1,13 +1,10 @@
 """ Run some tests and generate warnings for proton configuration issues
 """
 
-import shutil
-import os
-import subprocess
 from .logger import log
 
 
-def esync_file_limits():
+def esync_file_limits() -> bool:
```

Comment:
Hmm if we're adding types to non-gamefixes, type annotations should probably be enforced by the linter, Pylint, and caught in the CI system to avoid having to add the missing type in the future. This can be addressed in a separate PR though

See https://pypi.org/project/flake8-annotations/

Reply:
We could also enforce types for the gamefixes.
This should be trivial, as most of them only consist of the `main()` function.

Reply:
We could and it'll be keep things consistent. Though, I don't think it'll add much though so you don't need to. 

For the game fixes, what we really want to catch from the linter are the typos, logical errors and some bad practices. There were just a ton of them before I added the linter in this project, and for the subset of fixes that return a non-None type for some reason I don't think it'll affect the behavior of the program

Reply:
Sure, I think it is just easier to maintain one linter configuration than multiple. Alternatively there might be an option to allow functions that return `None` to have no type. Not sure about that though.